python/server/qgsserverprojectutils.sip

```diff
@@ -111,12 +111,90 @@ namespace QgsServerProjectUtils
       */
     int wmsMaxHeight( const QgsProject &project );
 
+    /** Returns if layer ids are used as name in WMS.
+      * \param project the QGIS project
+      * \returns if layer ids are used as name.
+      */
+    bool wmsUseLayerIds( const QgsProject &project );
+
+    /** Returns if the info format is SIA20145.
+      * \param project the QGIS project
+      * \returns if the info format is SIA20145.
+      */
+    bool wmsInfoFormatSia2045( const QgsProject &project );
+
+    /** Returns if Inspire is activated.
+      * \param project the QGIS project
+      * \returns if Inspire is activated.
+      */
+    bool wmsInspireActivated( const QgsProject &project );
```

Comment:
A small suggestion, can you maybe rename this to `wmsInspireActive` ? I think we are using the `ed` form for slots. The comment should also become `Return if Inspire is active`